cmd/shcd_test.go

```diff
@@ -18,123 +18,35 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-const _schemaFile = "../internal/files/shcd-schema.json"
+const _schema = "shcd-schema.json"
 
+var _schemaFile = filepath.Join("../internal/files", _schema)
+
+var switch_meta_expected = "../testdata/expected/" + switch_metadata
+var hmn_conn_expected = "../testdata/expected/" + hmn_connections
+var app_node_expected = "../testdata/expected/" + application_node_config
```

Comment:
When do these testdata files get created?

Reply:
They are part of this PR.  They care made from the valid JSON file